eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md

```diff
@@ -0,0 +1,168 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- azure-event-hubs
+description: "Azure Spring Cloud Sample project for Event Hub Integration client library"
+urlFragment: "azure-spring-integration-eventhubs-sample"
+---
+
+# Spring Cloud Azure Event Hubs Integration Code Sample shared library for Java
+
+## Key concepts
+
+This sample demonstrates how to use Spring Integration for Azure
+Event Hubs.
+
+
+## Getting started
+
+Running this sample will be charged by Azure. You can check the usage and bill at
+[this link][azure-account].
+
+
+### Create Azure resources
+
+1. Create [Azure Event Hubs][create-event-hubs].
+    After creating the Azure Event Hubs, you
+    can create your own Consumer Group or use the default "$Default" Consumer Group.
+
+2. Create [Azure Storage][create-azure-storage] for checkpoint use.
+
+### Configuration credential options
+
+We have several ways to config the Spring Integration for Service
+Bus. You can choose anyone of them.
+
+>[!Important]
+>
+>  When using the Restful API to send messages, the **Active profiles** must contain `manual`.
+> 
+#### Method 1: Connection string based usage
+
+1. Update [application.yaml].
```

Comment:
no link here

Reply:
updated to keep consistent with the other parts.